pkgs/applications/misc/octoprint/plugins.nix

```diff
@@ -367,6 +534,25 @@ in
     };
   };
 
+  timelapsepurger = buildPlugin rec {
+    pname = "firmwareupdater";
+    version = "0.1.4";
+
+    src = fetchFromGitHub {
+      owner = "jneilliii";
+      repo = "OctoPrint-TimelapsePurger";
+      rev = version;
+      sha256 = "sha256-XS4m4KByScGTPfVE4kuRLw829gNE2CdM0RyhRqGGxyw=";
+    };
+
+    meta = with lib; {
+      description = "Automatically deletes timelapses that are older than configured timeframe.";
```

Comment:
```suggestion
      description = "Automatically deletes timelapses that are older than configured timeframe";
```

Reply:
Would like to keep it consistent with the other plugins, as well as keeping it the same as the official plugin's repo